README.md

```diff
@@ -2,42 +2,17 @@
 
 # Ember-Electron
 
-![Latest release on NPM](https://img.shields.io/npm/v/ember-electron.svg) [![Build status](https://ci.appveyor.com/api/projects/status/5rhwhar361uad07v?svg=true)](https://ci.appveyor.com/project/adopted-ember-addons/ember-electron)
- ![Travis CI Build Status](https://secure.travis-ci.org/adopted-ember-addons/ember-electron.svg?branch=master) ![Code Climate](https://codeclimate.com/github/adopted-ember-addons/ember-electron.svg)
+![Latest release on NPM](https://img.shields.io/npm/v/ember-electron.svg) 
+[![Build status](https://ci.appveyor.com/api/projects/status/5rhwhar361uad07v?svg=true)](https://ci.appveyor.com/project/adopted-ember-addons/ember-electron)
+![Travis CI Build Status](https://secure.travis-ci.org/adopted-ember-addons/ember-electron.svg?branch=master) 
+![Code Climate](https://codeclimate.com/github/adopted-ember-addons/ember-electron.svg)
```

Comment:
This 404s -- were you going to set up code climate?

Reply:
This was existing. We can remove, if you want.

Reply:
Oh yeah, I see -- mis-read the diff. Well, let's pull it since it 404s.